docs/DeveloperGuide.md

```diff
@@ -1,29 +1,104 @@
-# Developer Guide
+# Developer Guide - Voyagers
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+This project is a team effort by the NUS CS2113 Group W14-4
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+The Voyagers application is designed as a command-line interface 
+(CLI) tool for managing travel trips. 
+It is implemented in Java and follows a modular architecture to promote scalability and maintainability.
 
+The following UML sequence diagram shows the flow of the application when the user enters a command.
+
+![Sequence Diagram](diagrams/seq_diagram_flow.png)
```

Comment:
<img width="250" alt="image 4" src="https://github.com/nus-cs2113-AY2324S2/tp/assets/111105911/2780afdd-dfe2-45b1-b95e-676d8a476f4e">

Should there be an additional operation after this arrow (before command object is deleted) due to the c.isExit() operation that is called?